app/scripts/modules/core/src/presentation/forms/layouts/ResponsiveFieldLayout.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import { HelpContextProvider } from 'core/help';
+
+import { IFieldLayoutProps } from '../interface';
+
+import '../forms.less';
+
+export class ResponsiveFieldLayout extends React.Component<IFieldLayoutProps> {
```

Comment:
How is this guy different than the standard field layout?  Could we add the responsiveness of this layout to the standard layout?

Reply:
This one's using @gcomstock's new CSS. I figured it'd be easier to just cutover than to try to make one that's compatible with both, but I can give that a try and see if it just works™